.travis.yml

```diff
@@ -1,12 +1,20 @@
 language: php
 
+addons:
+  apt:
+    update: true
+    packages:
+      - libwebp-dev
 php:
   - 5.5
   - 5.6
   - 7.0
   - 7.1
   - 7.2
 
+before_install:
```

Comment:
The Travis CI build is failed because the `configure` file is not found.

It seems that you want to compile the PHP with gd extension again.

The `configure` file should be in the `php-src` folder and I don't think this folder will be existed in this current working directory.

Reply:
Hi, yes I was trying to understand how to enable webp support